frontend/src/scenes/notebooks/Notebook/notebooksListLogic.ts

```diff
@@ -44,16 +48,33 @@ export const openNotebook = (
     }
 
     popoverLogic?.actions.setInitialAutofocus(focus)
+
+    const theMountedNotebookLogic = notebookLogic({ shortId: notebookId })
+
+    // there's a race here between opening up the notebook and mounting its logic
+    // and that logic having an editor ready to receive commands
+    let retryCount = 0
+    while (theMountedNotebookLogic.values.editor === null && retryCount < 100) {
+        await new Promise((r) => setTimeout(r, 10))
+        retryCount += 1
+    }
```

Comment:
I wonder if there is something we could do here like calling an action or something to say "notebookLogic(...).actions.onReady(cb)` which means we don't need any "hacky" waiting type things?